### DISCUSSION TITLE
How can I exec / ssh into the VM?

### DISCUSSION THREAD

**Discussion by rsmoorthy** [rancher-sandbox/rancher-desktop#1313] [Q&A] [answered]
Hi,

I have been trying to exec or ssh into the VM, but so far I did not find any ways to do it. I suppose it is running lima, but running limactl list or limactl shell is not working either.

Anyone?

**Comment by jandubois** [comment, marked as answer]
You can use this command (long line, embedded spaces) to create a shell inside the VM:

```shell
LIMA_HOME="$HOME/Library/Application Support/rancher-desktop/lima" "/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl" shell 0
```

If you break anything, you get to keep all the pieces! 😸

**Reply by rsmoorthy** [reply]
Thanks. Works great!
